### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by AbdominalbitMaster** [open-spaced-repetition/fsrs4anki#331]
**Research**

*Enter an [x] character to confirm the points below:*

- [x] I have checked the [FAQ](https://github.com/open-spaced-repetition/fsrs4anki/wiki/FAQ) and could not find an answer to my question
- [x] I have read the [wiki](https://github.com/open-spaced-repetition/fsrs4anki/wiki) and still felt confused
- [x] I have searched for similar existing questions here

**Question**
Hello!

**Comment by Expertium** [comment]
No, I don't think this is compatible with FSRS. However, this is similar to an idea I proposed some time ago, so in the future something similar will likely be implemented.

**Comment by AkiraChisaka** [comment]
My assumption is a random guess, but I would say it's probably impossible to be compatible, since they are both interval modifying algorithms.

It's probably better to just hope this feature becomes more demanded and coded into FSRS one day.

**Comment by L-M-Sherlock** [comment]
I think the features implemented in the priority add-on is equivalent to setting specific requested retention for a specific card.

**Comment by Expertium** [comment]
You can set your desired retention in the scheduler:
`"requestRetention": 0.8,`
It sets this level for all cards within a deck that uses this set of parameters. Currently, you cannot set different levels of desired retention for individual cards.

**Comment by user1823** [comment]
> ... tells me to go look at https://github.com/open-spaced-repetition/fsrs4anki/wiki/Free-Spaced-Repetition-Scheduler but I still can't find it!

Retrievability is the same as retention.

**Comment by L-M-Sherlock** [comment]
Fine. The answer to this question is `incompatible`. And I recommend moving those low priority cards into another deck and setting a low retention.

**Comment by AkiraChisaka** [comment]
> And I recommend moving those low priority cards into another deck and setting a low retention.

Hun, I never thought about this hack. That should work perfectly, thanks!
